core/templates/pages/learner-dashboard-page/goals-tab.component.css

```diff
@@ -5,12 +5,14 @@
 */
 .oppia-arrow-button {
   color: #00645c;
+  cursor: pointer;
```

Comment:
Should we also show a pointer when hovering over the "Learn Fractions" text part? It looks like the text is clickable too.